### PR TITLE
Remove 'github' from the secrets bucket SSH key name

### PR DESCRIPTION
**Pull request by toolmantim**
There's nothing Github specific about this key at all, so let's not include it in the name

### CODE REVIEW

**Comment by lox** [comment]
Is this a breaking change for existing installs?

**Comment by toolmantim** [comment]
Yep, but I was thinking of adding a fallback

**Comment by toolmantim** [comment]
@lox so I've added a fallback for people still on the `_github` version, along with support for a top level SSH key for people when per-pipeline is overkill. How's that look?

File: README.md [hunk 1]
```diff
@@ -2,29 +2,30 @@
...
+* CloudWatch metrics from the Buildkite API
+* Support for stable, unstable or experimental Buildkite Agent releases
 
 ## Getting Started
 
 The easiest way is to launch the latest built version via this button:
 
 [![Launch Buildkite AWS Stack](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/images/cloudformation-launch-stack-button.png)](https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=buildkite&templateURL=https://s3.amazonaws.com/buildkite-aws-stack/aws-stack.json)
 
+> Although the stack will create it's own VPC by default, we highly recommend following best practice by setting up a separate development AWS account and using role switching and consolidated billing—see the [Delegate Access Across AWS Accounts tutorial](http://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html) for more information.
```
Comment (lox) [inline README.md]: 👌

File: README.md [hunk 2]
```diff
@@ -49,42 +50,73 @@ make create-stack
...
 
-You should encrypt your objects with a project-specific key and provide it in `BUILDKITE_SECRETS_KEY` which will be used to decrypt all the files found in the secrets bucket.
+## Secrets Bucket Support
 
-### Creating a new project
+The stack has a `SecretsBucket` parameter which will allow your build agents to automatically get access to SSH private keys and environment hooks for exposing environment variables to builds. The stack doesn't create the bucket for you, you need to do this yourself, but it does create a role that gives read access to the build machines. 
+
+The secrets bucket can contain the following files:
+
+* `/id_sra` - An optional private key to use Git SSH operations when there is no pipeline-specific key present
```
Comment (lox) [inline README.md]: Typo?

**Comment by lox** [comment]
Thoughts on swapping to something like `ssh_private_key` rather than `id_rsa`? It could very well be dsa or ECC.

**Comment by toolmantim** [comment]
Yeah that's a good idea. It's not even really specific to git is it? `ssh_key` could be fine too?

Also I noticed a lot of people use netrc and GitHub API keys with read repo instead of SSH keys. I had pondered how to support that… some other PR, just figured I'd plant the seed

File: README.md (hunk 2 above)
Comment (toolmantim) [inline README.md]: Whoops

**Comment by toolmantim** [comment]
Ok, changed it to `private_ssh_key` and fixed the Readme. Will merge when green…

**Pull request merged**